application/src/main/kotlin/Main.kt

```diff
@@ -1,33 +1,156 @@
-import androidx.compose.material.MaterialTheme
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.*
+import androidx.compose.runtime.*
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.window.Window
 import androidx.compose.ui.window.WindowPosition
 import androidx.compose.ui.window.application
 import androidx.compose.ui.window.rememberWindowState
 import models.SettingsModel
+import models.utils.GraphInfo
+import themes.JetCorners
+import themes.JetFontFamily
+import themes.JetSize
+import themes.JetStyle
 import views.MainScreen
 import viewmodels.MainScreenViewModel
 import java.awt.Dimension
+import java.io.File
+import java.io.FileWriter
 
 /**
  * A pair representing the initial size of the application window.
  */
 val windowSizeStart = Pair(1000f, 700f)
 
+/**
+ * Data class representing the settings for application.
+ *
+ * @property currentStyle The current style of application, represented by a MutableState of [JetStyle]
+ * @property currentFontSize The current font size of text in application, represented by a MutableState of [JetSize]
+ * @property currentCornersStyle The current corner style of application, represented by a MutableState of [JetCorners]
+ * @property currentFontFamily The current font family of text in application, represented by a MutableState of [JetFontFamily]
+ * @property isDarkMode A boolean MutableState indicating whether the application is in dark mode or not
```

Comment:
У нас в проекте принято, что описание свойств начинается с маленькой буквы.

```suggestion
 * @property currentStyle the current style of application, represented by a MutableState of [JetStyle]
 * @property currentFontSize the current font size of text in application, represented by a MutableState of [JetSize]
 * @property currentCornersStyle the current corner style of application, represented by a MutableState of [JetCorners]
 * @property currentFontFamily the current font family of text in application, represented by a MutableState of [JetFontFamily]
 * @property isDarkMode a boolean MutableState indicating whether the application is in dark mode or not
```

Reply:
Исправил